packages/devtools_app/lib/src/flutter/common_widgets.dart

```diff
@@ -319,19 +319,27 @@ class BulletSpacer extends StatelessWidget {
 /// A small element containing some accessory information, often a numeric
 /// value.
 class Badge extends StatelessWidget {
-  const Badge({@required this.child});
+  const Badge({@required this.text});
```

Comment:
nit: make text be a required parameter instead of a required named parameter.
That would be consistent with widgets such as Text and save a bit of typing wherever this is used.